### DISCUSSION TITLE
Settings Avatar full name makes the menu large

### DISCUSSION THREAD

**Discussion by voodoo** [AllYourBot/hostedgpt#546] [General]
However long your name is,  the nav will be that wide. The truncate tailwind css is not doing anything ?

<img width="349" alt="Screenshot 2024-11-13 at 3 50 19 PM" src="https://github.com/user-attachments/assets/18ef5fd8-d1e8-42ce-ab17-1351b1e19857">

Here I am using ruby truncate to 20.

<img width="298" alt="Screenshot 2024-11-13 at 3 53 39 PM" src="https://github.com/user-attachments/assets/a1ce1714-58e2-4d67-bce9-1ea5db9b6a3e">

I dont know if that is a fix that needs fixing, what do you think?

**Comment by keithschacht** [comment]
It'd be great to fix this. Want to tee up a PR for it?

**Comment by voodoo** [comment]
Yes sir, yes sir! :)
